src/Microsoft.AspNetCore.Razor.Design/Microsoft.AspNetCore.Razor.Design.csproj

```diff
@@ -89,16 +86,41 @@
     <MSBuild Projects="@(ToolProject)" Properties="PublishDir=$(MSBuildProjectDirectory)\$(OutputPath)tools\" Targets="Publish" />
 
     <ItemGroup>
-      <_RazorTool Include="$(OutputPath)tools\**\*" Exclude="$(OutputPath)tools\**\*.xml;$(OutputPath)tools\**\*.pdb" />
-    </ItemGroup>
-
-    <ItemGroup>
-      <None Include="@(_RazorTool)">
-        <PackagePath>tools\%(RecursiveDir)%(Filename)%(Extension)</PackagePath>
-        <Pack>true</Pack>
-      </None>
+      <_RazorTool Include="$(OutputPath)tools\**\*" />
     </ItemGroup>
 
     <Error Text="_RazorTool is empty. This is a bug" Condition="'@(_RazorTool)'==''" />
   </Target>
+
+  <Target Name="PopulateNuspec" BeforeTargets="GenerateNuspec" DependsOnTargets="BuiltProjectOutputGroup;DebugSymbolsProjectOutputGroup;_BuildDependencyProjects">
+
+    <PropertyGroup>
+      <!-- Make sure we create a symbols.nupkg. -->
+      <IncludeSymbols>true</IncludeSymbols>
+
+      <NuspecProperties>
+        id=$(MSBuildProjectName);
+        version=$(PackageVersion);
+        authors=$(Authors);
+        description=$(Description);
+        tags=$([MSBuild]::Escape($(PackageTags)));
```

Comment:
Nit: tags in nuspec should be space separated, so `$(PackageTags.Replace(';', ' '))`